splunk_eventgen/lib/plugins/generator/perdayvolumegenerator.py

```diff
@@ -31,17 +31,15 @@ def gen(self, count, earliest, latest, samplename=None):
 
         # Create a counter for the current byte size of the read in samples
         currentSize = 0
-        # Replace event tokens before calculating the size of the event
-        updated_sample_dict = GeneratorPlugin.replace_tokens(self, self._sample.sampleDict, earliest, latest)
 
```

Comment:
Use the raw sample to calculate the size of event and replace the token later in `build_events`.